pkg/utils/k8sutil.go

```diff
@@ -790,8 +700,6 @@ func EnsureFuncDeployment(client kubernetes.Interface, funcObj *spec.Function, o
 		// update deployment for loading dependencies
 		langruntime.UpdateDeployment(dpm, runtimeVolumeMount.MountPath, funcObj.Spec.Runtime)
 	}
-	//TODO: remove this when init containers becomes a stable feature
```

Comment:
if you remove this Kubernetes 1.7 becomes unusable, isn't it?

Reply:
No it is not. 

https://travis-ci.org/kubeless/kubeless/jobs/327658287